raiden_contracts/CHANGELOG.md

```diff
@@ -0,0 +1 @@
+../CHANGELOG.md
```

Comment:
I can't see it in the diff, but I assume that this is indeed a symlink, not a file with the content `../CHANGELOG.md`.